content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/refresh.rb

```diff
@@ -13,9 +14,18 @@ def initialize(handle = $evm)
               end
 
               def main
-                @handle.log("info", "Starting Refresh")
-                service.refresh(@handle.root["service_action"])
-                @handle.log("info", "Ending Refresh")
+                @handle.log(:info, "Starting Refresh")
+
+                begin
+                  service.refresh(@handle.root["service_action"])
```

Comment:
@mkanoor @billfitzgerald0120 Madhu and I discussed that all of the generic state machine methods should validate the service_action. We're going to follow up with a PR for the method and spec changes.